spec/pQueue.js

```diff
@@ -0,0 +1,122 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import PriorityQueue from '../src/pQueue'
+
+chai.use(chaiChange)
+
+describe('PriorityQueue', () => {
+  it('exists', () => {
+    expect(PriorityQueue).to.be.a('function')
+  })
+
+  context('enqueue', (element, priority) => {
+    it('adds one element to the queue queue.', () => {
+      const pQueue = new PriorityQueue
```

Comment:
Rather than declaring a new instance of pQueue in every test, you can use a beforeEach hook and only have to write it once: https://mochajs.org/#hooks